apigenerator/resources/1 - Project/2 - Database/database_conn_files/pgsql/ssl_authentication/PgSqlConnection.py

```
@@ -19,8 +19,7 @@ def get_pgsql_connection_schema_internet():
                                                  + get_global_variable('pgsql_password') + '@' \
                                                  + get_global_variable('ssl_hostname') + ':' \
                                                  + get_global_variable('pgsql_port') + '/' \
-                                                 + get_global_variable('pgsql_database_name') + '?options=-c%20search_path=' \
-                                                 + get_global_variable('pgsql_schema') + ',public' + '?' \
+                                                 + get_global_variable('pgsql_database_name') + '?' \
```

Ruff (0.11.9)

`get_global_variable` may be undefined, or defined from star imports

(F405)